boards/arm/nrf52840_pca10056/nrf52840_pca10056.dts

```diff
@@ -201,6 +201,15 @@ arduino_spi: &spi3 {
 	mosi-pin = <45>;
 };
 
+&i2s0 {
```

Comment:
What about other SoCs/Boards with I2S interface?

Reply:
This is easily added for the nRF52832 at least. We have done this, and it works fine. Though, we have not run in depth tests on this.